pkg/tagger/local/clock.go

```diff
@@ -0,0 +1,21 @@
+package local
+
+import "time"
+
+type clock interface {
+	Now() time.Time
+}
+
+type realClock struct{}
+
+func (realClock) Now() time.Time {
+	return time.Now()
+}
+
+type fakeClock struct {
+	now time.Time
+}
+
+func (f fakeClock) Now() time.Time {
+	return f.now
+}
```

Comment:
if it is only for test purpose (I suppose), Could you move it into `clock_test.go` file and `package local_test` package

Reply:
it seems to be used only in `pkg/tagger/local/tagstore_test.go` so maybe you can move it in this file directly